charts/quantum-serverless/charts/gateway/values.yaml

```diff
@@ -46,11 +41,41 @@ application:
     channel: "ibm_quantum"
     url: "https://auth.quantum-computing.ibm.com/api"
 
-database:
-  host: postgresql
-  port: 5432
-  user: bn_keycloak
-  name: bitnami_keycloak
+cos:
+  claimName: gateway-claim
+
+secrets:
+  secretKey:
+    create: true
+    name: gateway-secret-key
+    key: django-secret-key
+    value: "django-insecure-&)i3b5aue*#-i6k9i-03qm(d!0h&662lbhj12on_*gimn3x8p7"
+  servicePsql:
+    create: true
+    name: service-psql-binding
+    key:
+      databaseName: database-name
+      databasePassword: database-password
+      host: database-host
+      port: database-port
+      userName: user-name
```

Comment:
These keys are not necessary in the values.yaml file.  They are hardcoded in the secret yaml file.

Reply:
They are for us because in our environment we don't create all the secrets and we don't have control over the key names (that's the reason because I added the `create` field to create or not the secret when deploying).

Reply:
To put an example this is the `key` for the `host` secret in our internal environment:

```
connection.postgres.hosts.0.hostname
```

Reply:
So my idea is, if you create the secrets with our scripts perfect. If not for whatever reason, you have a way to not install them and specify those custom secrets (as it is our case).

Reply:
I accept other approach if someone knows a better one, btw. This was just the easiest way to handle the problem.

Reply:
I'm not sure how these are used for the other environment but if the secret is defined like this
```
stringData:
  {{ .Values.secrets.servicePsql.key.databaseName }}: {{ .Values.secrets.servicePsql.value.databaseName }}
  {{ .Values.secrets.servicePsql.key.databasePassword }}: {{ .Values.secrets.servicePsql.value.databasePassword }}
  {{ .Values.secrets.servicePsql.key.host }}: {{ .Values.secrets.servicePsql.value.host }}
  {{ .Values.secrets.servicePsql.key.port }}: {{ .Values.secrets.servicePsql.value.port }}
  {{ .Values.secrets.servicePsql.key.userName }}: {{ .Values.secrets.servicePsql.value.userName }}
```
instead of 
```
stringData:
  database-name: {{ .Values.secrets.servicePsql.value.databaseName }}
  database-password: {{ .Values.secrets.servicePsql.value.databasePassword }}
  database-host: {{ .Values.secrets.servicePsql.value.host }}
  database-port: {{ .Values.secrets.servicePsql.value.port }}
  user-name: {{ .Values.secrets.servicePsql.value.userName }}
```
then it makes sense.

Reply:
oh wait, is that possible? It gave me an error when I tried but maybe I did something wrong

Reply:
@akihikokuroda thank you for the feedback, it worked, i don't know what problem i had previously. Let me know if you see something more weird 🙏 

Solved in [4300cfb](https://github.com/Qiskit-Extensions/quantum-serverless/pull/928/commits/4300cfb41763f58ae58ee0bf72d8850cda71d901)